core/web/resolver/mutation.go

```diff
@@ -913,3 +922,59 @@ func (r *Resolver) DeleteChain(ctx context.Context, args struct {
 
 	return NewDeleteChainPayload(&chain, nil), nil
 }
+
+func (r *Resolver) CreateJob(ctx context.Context, args struct {
+	Input struct {
+		TOML string
+	}
+}) (*CreateJobPayloadResolver, error) {
+	if err := authenticateUser(ctx); err != nil {
+		return nil, err
+	}
+
+	jbt, err := job.ValidateSpec(args.Input.TOML)
+	if err != nil {
+		return NewCreateJobPayload(nil, map[string]string{
+			"TOML spec": errors.Wrap(err, "failed to parse TOML").Error(),
+		}), nil
+	}
+
+	var jb job.Job
+	config := r.App.GetConfig()
+	switch jbt {
+	case job.OffchainReporting:
+		jb, err = offchainreporting.ValidatedOracleSpecToml(r.App.GetChainSet(), args.Input.TOML)
+		if !config.Dev() && !config.FeatureOffchainReporting() {
+			return nil, errors.New("The Offchain Reporting feature is disabled by configuration")
+		}
+	case job.DirectRequest:
+		jb, err = directrequest.ValidatedDirectRequestSpec(args.Input.TOML)
+	case job.FluxMonitor:
+		jb, err = fluxmonitorv2.ValidatedFluxMonitorSpec(config, args.Input.TOML)
+	case job.Keeper:
+		jb, err = keeper.ValidatedKeeperSpec(args.Input.TOML)
+	case job.Cron:
+		jb, err = cron.ValidatedCronSpec(args.Input.TOML)
+	case job.VRF:
+		jb, err = vrf.ValidatedVRFSpec(args.Input.TOML)
+	case job.Webhook:
+		jb, err = webhook.ValidatedWebhookSpec(args.Input.TOML, r.App.GetExternalInitiatorManager())
+	default:
+		return NewCreateJobPayload(nil, map[string]string{
```

Comment:
This seems important enough that should be returned as a payload error

Reply:
After further looking at this, the `job.ValidateSpec` also validates the job type, so really this never gets called... so I guess we still return some specific error, but it will never get executed @jkongie